EvidenceApi/V1/Gateways/NotifyGateway.cs

```diff
@@ -128,7 +129,8 @@ private static string GetTemplateIdFor(DeliveryMethod deliveryMethod, Communicat
                     {"resident_name", resident.Name},
                     {"reason", evidenceRequest.Reason},
                     {"magic_link", MagicLinkFor(evidenceRequest)},
-                    {"note_to_resident", evidenceRequest.NoteToResident}
+                    {"note_to_resident", evidenceRequest.NoteToResident},
+                    {"document_types", documentTypes}
```

Comment:
Is this the best parameter name to use in Gov Notify? Should I simplify it to 'documents'?

Reply:
👍